libraries/app/include/graphene/app/api.hpp

```diff
@@ -176,10 +176,32 @@ namespace graphene { namespace app {
                                                                         uint32_t stop = 0,
                                                                         unsigned limit = 100,
                                                                         uint32_t start = 0) const;
-
+         /**
+          * @brief Get history of orders with b:a pair
+          * @param a Base in pair
+          * @param b Quote in pair
+          * @param limit Maximum number of orders to retrieve
+          * @return A history of orders with b:a pair, where a <= b, otherwise a and b will be swaped
```

Comment:
If always a <= b then it doesn't make sense to call a "Base" and b "Quote". Backend is ignorant of UI market order.